test/recipes/15-test_genec.t

```diff
@@ -197,6 +198,20 @@ ok(run(app([ 'openssl', 'genpkey',
              '-text'])),
     "generate a private key and serialize it using the base provider");
 
+ok(run(app([ 'openssl', 'genpkey',
+             '-algorithm', 'EC',
+             '-pkeyopt', 'ec_paramgen_curve:P-256',
+             '-pkeyopt', 'ec_param_enc:explicit',
+             '-text'])),
+   "generate a key with explicit encoding");
+
+ok(run(app([ 'openssl', 'genpkey',
+             '-algorithm', 'EC',
+             '-pkeyopt', 'ec_paramgen_curve:P-256',
+             '-pkeyopt', 'ec_param_enc:named_curve',
+             '-text'])),
+   "generate a key with named_curve encoding");
+
```

Comment:
Nit: These 2 tests are a bit superfluous: just after them we test again the same thing for all supported curves!

Reply:
Good point.  I'll remove this, then

Reply:
Done